src/components/simplified-exchange-component.ts

```
@@ -163,11 +163,19 @@ export class SimplifiedExchangeComponent {
     const yourTokenGroup = document.getElementById("yourTokenGroup") as HTMLOptGroupElement;
     const otherTokenGroup = document.getElementById("otherTokenGroup") as HTMLOptGroupElement;
 
+    // Minimum USD value to display a token in the inventory list
+    const MIN_USD_VALUE = 1.0;
+
     if (refreshData?.tokenBalances) {
+      // Filter out dust tokens (value < $1.00)
+      const significantBalances = refreshData.tokenBalances.filter(
+        (balance) => !balance.usdValue || balance.usdValue >= MIN_USD_VALUE
+      );
```

ESLint

[error] Replace `⏎········(balance)·=>·!balance.usdValue·||·balance.usdValue·>=·MIN_USD_VALUE⏎······` with `(balance)·=>·!balance.usdValue·||·balance.usdValue·>=·MIN_USD_VALUE`

(prettier/prettier)